[fight_flow_0]: HelloWorld pr for user yj-wang

yjwang346 opened the pull request:
<!--
Thanks for opening a pull request!
If this is your first pull request you can find detailed information on how 
to contribute here:
  * [贡献规范](https://github.com/datawhalechina/whale-starry/blob/main/stl/README.md#4%E8%A7%84%E8%8C%83)

Rename your pr title:
    [Team_Task]: Your Custom Title.

-->

### What?

<!--
- brief: 描述本次pr的内容简介
- author: 作者 
-->

### What changes are included in this PR?

<!--
当前pr包含了哪些文件变更?
-->

### Are these changes tested?

<!--
是否对编写的代码进行过测试？
或者
是否对学习的开源项目测试？
-->

Light-City commented:
> ### What?
> ### What changes are included in this PR?
> ### Are these changes tested?

content is emtpy?

Light-City commented on stl/fight_flow/yj-wang/yjwang/tempCodeRunnerFile.exe:
remove it or add into .gitignore.

yjwang346 replied:
sorry, I'll do it tomorrow for a better version. thanks

Light-City commented:
HelloWorld -> hello_world, lowercase underscore

Light-City commented:
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

yjwang346 commented:
[new version] Could you see the new version I modified? I changed teh structure to meet the request, remove the .exe file and make the file renamed. (sorry for that I have planed to modified regardless of your work load.) 
*********************************
What?
- brief: 这是一个测试pr，尝试按照模板内容提交
- author: yjwang346

What changes are included in this PR?
包含了一个团队目录、个人目录、个人学习内容。

Are these changes tested?
测试代码见main.cpp，里面使用cout打印了一个"Hello World!"语句

martinx commented:
Please try to use markdown to write your content.

```
## What?

* brief: 这是一个测试pr，尝试按照模板内容提交
* author: yjwang346

## What changes are included in this PR?

包含了一个团队目录、个人目录、个人学习内容。

## Are these changes tested?

测试代码见main.cpp，里面使用cout打印了一个"Hello World!"语句
```

yjwang346 commented:
OK. I misunderstood. And how about now?
